packages/core/src/ceramic.ts

```diff
@@ -655,6 +656,47 @@ export class Ceramic implements StreamReaderWriter, StreamStateLoader {
     )
   }
 
+  /**
+   * Waits for Model used to publish NodeMetrics to be available locally.
+   * Since we subscribe to the metamodel at startup, so long as some connected node on the network
+   * has the model, it should eventually be available locally.
+   * @param model
+   */
+  async _waitForMetricsModel(model: StreamID): Promise<void> {
+    let attemptNum = 0
+    let backoffMs = 100
+    const maxBackoffMs = 1000 * 10
+    const delay = async function (ms) {
+      return new Promise<void>((resolve) => setTimeout(() => resolve(), ms))
+    }
+
+    // eslint-disable-next-line no-constant-condition
+    while (true) {
```

Comment:
note we will wait forever if the node config says it wants to publish node metrics but the model doesn't ever show up (e.g. because the node is isolated from the rest of the network, or connected to a local network where the Model isn't available)

Reply:
yeah that is bad we should just continue with our work

there are way too many ways for ceramic nodes to fail, i would strongly prefer NodeMetrics and observability NOT be a way that nodes error out and don't start.

Reply:
I think there's an argument to be made here that if the user asked for metrics in the config and we fail, they should learn that and choose to turn off metrics or to fix the underlying error.

But that said I don't feel strongly and since we plan to turn this on by default, I see the argument to minimize it's impact.  I'll update to make this time out without failing startup